__tests__/app.test.js

```diff
@@ -75,3 +75,26 @@ describe("task 3. GET /api", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("GET 200. should return an array of all articles, sorted by DESC (earliest if the first), has no body as property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(articleData.length);
+        expect(articles[0].article_id).toBe(3);
+        articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+        });
+      });
+  });
+  test("GET 404. url not found", () => {
```

Comment:
If a URL is misspelled it doesn't matter how close it is to the intended url the fact it doesn't match means it will be caught in the same way as any other misspelling so this only needs to be tested once on a server